tests/UI/campaigns/functional/API/01_clientCredentialGrantFlow/02_externalAuthServer/01_authorizationEndpoint.ts

```diff
@@ -1,164 +1,46 @@
 
 import {expect} from 'chai';
-import {APIRequestContext, request} from 'playwright';
+import {APIRequestContext} from 'playwright';
 
 const baseContext: string = 'functional_API_clientCredentialGrantFlow_externalAuthServer_authorizationEndpoint';
 
-// @todo : https://github.com/PrestaShop/PrestaShop/issues/33946
 describe('API : External Auth Server - Authorization Endpoint', async () => {
-  let apiContextBO: APIRequestContext;
   let apiContextKeycloak: APIRequestContext;
-  let accessTokenKeycloak: string;
 
   before(async () => {
-    apiContextBO = await request.newContext({
-      baseURL: global.BO.URL,
-      // @todo : Remove it when Puppeteer will accept self signed certificates
-      ignoreHTTPSErrors: true,
-    });
-    if (!global.GENERATE_FAILED_STEPS) {
-      /*
-      apiContextKeycloak = await request.newContext({
-        baseURL: global.keycloakConfig.keycloakExternalUrl,
-        // @todo : Remove it when Puppeteer will accept self signed certificates
-        ignoreHTTPSErrors: true,
-      });
-
-      accessTokenKeycloak = await keycloakHelper.createClient(
-        global.keycloakConfig.keycloakClientId,
-        'PrestaShop Client ID',
-        false,
-        true,
-      );
-      expect(accessTokenKeycloak.length).to.be.gt(0);
-      */
-    }
-  });
-
-  after(async () => {
-    if (!global.GENERATE_FAILED_STEPS) {
-      /*
-      const isRemoved: boolean = await keycloakHelper.removeClient(global.keycloakConfig.keycloakClientId);
-      expect(isRemoved).to.eq(true);
-      */
-    }
+    apiContextKeycloak = await helper.createAPIContext(global.keycloakConfig.keycloakExternalUrl);
   });
 
   describe('Authorization Endpoint', async () => {
-    it('should request the endpoint /access_token with method GET', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'requestAuthWithMethodGET', baseContext);
-
-      this.skip();
-
-      const apiResponse = await apiContextBO.get('access_token');
-      expect(apiResponse.status()).to.eq(405);
-    });
-
-    it('should request the endpoint /access_token with method POST', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'requestAuthWithMethodPOST', baseContext);
-
-      this.skip();
-
-      const apiResponse = await apiContextBO.post('access_token');
-      expect(apiResponse.status()).to.eq(400);
-    });
-
-    it('should request the endpoint /access_token with method POST with unuseful data', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'requestAuthWithMethodPOSTUnusefulData', baseContext);
-
-      this.skip();
-
-      const apiResponse = await apiContextBO.post('access_token', {
-        form: {
-          notUsed: 'notUsed',
-        },
-      });
-      expect(apiResponse.status()).to.eq(400);
-    });
-
-    it('should request the endpoint /access_token with method POST with invalid data', async function () {
```

Comment:
this test and the others deleted in this file are just deleted and not ported anywhere, is this normal ?

Reply:
Yes the access token endpoint from keycloak is external, we only check here that it is correctly accessible and usable with the fixtures data We didn't develop this endpoint so it makes no sense to test it any further The previous tests were initially added because there as a gateway feature that was though about at some point, but it's a future feature (maybe), so the tests here were simplified

This as already discussed and validated with @Progi1984